docs/REST API/rest-api.md

```diff
@@ -6219,6 +6219,40 @@ Creates a script execution request and waits for a result to return (up to a 1 m
 }
 ```
 
+### Get Script Result
+
+_Available in Fleet Premium_
+
+Gets the result of a script.
+
+#### Parameters
+
+| Name         | Type   | In   | Description                                   |
+| ----         | ------ | ---- | --------------------------------------------  |
+| execution_id | string | path | **Required**. The execution id of the script. |
+
+#### Example
+
+`GET /api/v1/fleet/scripts/results/{execution_id}`
+
+##### Default Response
+
+`Status: 2000`
+
+```json
+{
+  "script_contents": "echo 'hello'",
+  "exit_code": 0 | null,
```

Comment:
```suggestion
  "exit_code": 0,
```
Rather than include both possibilities, let's just document what's expected for this example and add a note in a blockquote underneath explaining the circumstances for `exit_code` sometimes being `null`